snap/snapcraft.yaml

```diff
@@ -21,7 +21,7 @@ parts:
   connectedhomeip:
     plugin: nil
     build-environment:
-      - TAG: v1.2.0.1
+      - TAG: master
```

Comment:
Use this branch which includes a python fix for testing purpose

Reply:
I've created a separate branch for v1.2, so we can proceed with merging this (using master branch of SDK) into main.